ProjectManagmentSystem/src/Modules/Context/AuthContext.tsx

```diff
@@ -10,13 +10,15 @@ export const useAuth = () => {
 
 export default function AuthContextProvider(props: any) {
   const [loginUser, setLoginUser] = useState(null);
-
-
+  let requestHeaders = {Authorization:`Bearer ${localStorage.getItem('token')}`}
+  let baseUrl = 'https://upskilling-egypt.com:3003/api/v1'
 
   const getUserData = () => {
     const encodedToken: any = localStorage.getItem("token");
     const decodedToken: any = jwtDecode(encodedToken);
```

Comment:
Instead of decoding the token, we can call get current user API. It will include all information needed including the user's avatar that we will need in the navbar

Reply:
Done